tests/ui/consts/const-eval/stable-metric/evade-deduplication-issue-118612.rs

```diff
@@ -0,0 +1,24 @@
+//@ check-pass
+
+#![allow(long_running_const_eval)]
+
+//@ compile-flags: -Z tiny-const-eval-limit -Z deduplicate-diagnostics=yes
```

Comment:
Please keep all `//@` together at the top of the file; that's the usual style. This here makes it look like the comment is attached to the `const FOO` which is misleading.